code/datums/mutations.dm

```diff
@@ -287,11 +287,13 @@
 /datum/mutation/human/dwarfism/on_acquiring(mob/living/carbon/human/owner)
 	if(..())	return
 	owner.resize = 0.8
+	owner.pass_flags = PASSTABLE
```

Comment:
This sets the pass flags to this instead of just adding it, this should be: owner.pass_flags |= PASSTABLE